proposed/http-message.md

```diff
@@ -1402,9 +1404,11 @@ interface UriInterface
      * This method MUST retain the state of the current instance, and return
      * an instance that contains the specified query string.
      *
-     * If the query string is prefixed by "?", that character MUST be removed.
-     * Additionally, the query string SHOULD be parseable by parse_str() in
-     * order to be valid.
+     * The query string SHOULD be parseable by parse_str() in order to be
```

Comment:
I didn't add it and also wondered. It basically means that only a subset of query strings need to be supported. Because `parse_str('=bar')` doesn't work but would be a valid query string AFAIK.